### PR TITLE
Update `Chat console` commands

### PR DESCRIPTION
**Pull request by RoanH**
For relevant context, see the following issue: https://github.com/ppy/osu-stable-issues/issues/1121

Essentially this is just making the wiki article match in game behaviour. The `/msg [user] [message]` command does not exist, instead `/chat`, `/msg` and `/query` all just open a new private message tab to the given user. I decided to not remove documented working commands that are not listed in game (such as `/part` and `/join`). I also fixed a few consistency issues I spotted while updating the articles. 

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Client/Interface/Chat_console/en.md [hunk 1]
```diff
@@ -78,19 +78,18 @@ Press the number or click the bar to activate:
...
 | `/delfriend [user]` | Remove `[user]` from your friends list. | `/delfriend Amigo` | You are no longer friends with Amigo. |
-| `/away [message]` | Sets an away message (sent to users PMing you). Leave message blank to cancel. | `/away I am John Smith.` | You have been marked as being away: I am John Smith. When Amigo /msg John Where are you~? BanchoBot: I am John Smith. |
+| `/away [message]` | Sets an away message (sent to users PMing you). Leave the message blank to cancel. | `/away I am John Smith.` | You have been marked as being away: I am John Smith. When Amigo /msg John Where are you~? BanchoBot: I am John Smith. |
 | `/bb` | Sends a message to Bancho to execute a command like `!stats [user]` | `/bb !stats Uan` | \[15/11/12\] Stats for [Uan](https://osu.ppy.sh/users/147623): Score: 47,323,299,680 (#1) Plays: 176293 (lv102) Accuracy: 98.95% |
-| `/chat [user]` | Open a new chat tab with specified user. | `/chat Amigo` | (Amigo tab is opened) |
+| `/chat [user]` or `/msg [user]` or `/query [user]` | Opens a new chat tab with the specified user. | `/chat Amigo` | (Amigo tab is opened) |
```
Comment (Walavouchey) [inline wiki/Client/Interface/Chat_console/en.md:83]:
```suggestion
| `/chat [user]`, `/msg [user]`, or `/query [user]` | Opens a new chat tab with the specified user. | `/chat Amigo` | (Amigo tab is opened) |
```

**Review by Walavouchey — COMMENTED** [review]
seems like some of the translations don't translate "or" 🤔

**Comment by RoanH** [comment]
Some of the existing translations do not translate 'or' or some command arguments yeah. I've fixed what I saw if there was an existing translated version available, but I'm not a native speaker for the articles that are missing translations.

**Comment by TicClick** [comment]
if you are waiting for the translations to get reviewed it's gonna take a long while to collect feedback for 12 languages. I suggest that you roll them back and leave only these you are proficient at, if there are any

File: wiki/Client/Interface/Chat_console/ja.md [hunk 2]
```diff
@@ -78,14 +78,13 @@ osu!にログインしている各ユーザーは、拡張チャットにユー
 | `/delfriend [user]` | フレンドリストから`[user]`を削除します。 | `/delfriend Amigo` | You are no longer friends with Amigo. |
 | `/away [message]` | アウェイメッセージを設定します(自分にPMを送っているユーザーに送信されます)。キャンセルする場合は、メッセージを空白にします。 | `/away I am John Smith.` | You have been marked as being away: I am John Smith. When Amigo /msg John Where are you~? BanchoBot: I am John Smith. |
 | `/bb` | Banchoにメッセージを送り、`!stats [user]`のようなコマンドを実行させる。 | `/bb !stats Uan` | \[15/11/12\] Stats for [Uan](https://osu.ppy.sh/users/147623): Score: 47,323,299,680 (#1) Plays: 176293 (lv102) Accuracy: 98.95% |
-| `/chat [user]` | 指定したユーザーとの新しいチャットタブを開く。 | `/chat Amigo` | (Amigo tab is opened) |
+| `/chat [user]`,`/msg [user]`もしくは`/query [user]` | 指定したユーザーとの新しいチャットタブを開く。 | `/chat Amigo` | (Amigo tab is opened) |
```
[the remark this answers is not in the document]
Comment (RoanH) [reply wiki/Client/Interface/Chat_console/ja.md]: Oops, forgot to switch keyboard layout, thanks for fixing.

**Comment by CloneWith** [comment]
Since I am currently working on #9414 , you can just roll back changes on Chinese translations and let me update it then, thx 😸

**Review by TicClick — APPROVED** [review]
¯\\\_(ツ)\_/¯
in all honesty it's a minor change -- let's merge it as-is and accept future corrections instead

**Pull request merged**